packages/graphback/src/schema/schemaTemplate.ts

```diff
@@ -10,21 +10,21 @@ import { TargetContext, TargetType } from './targetSchemaContext';
 // }
 
 const inputs = (defs: TargetType[]): string => {
-  return `${defs.map((d: TargetType) => `input ${d.name}Input {
-    ${d.fields.join('\n    ')}
+  return `${defs.map((d: TargetType) => `${d.type} ${d.name}Input {
+    ${[...new Set(d.fields)].join('\n    ')}
```

Comment:
That is very creative way to remove duplicates. However we do not control what field takes precedence. Set will literally remove duplicate randomly which may be a problem if duplicate removal from the db will remove different one. We can end up with incompatible types. 

I'm not 100% sure about this as I could not replicate it, but basing on my understanding of the code this could happen.